package.json

```diff
@@ -39,12 +40,13 @@
     },
     "homepage": "https://github.com/ProtonMail/pmcrypto#readme",
     "dependencies": {
+        "comlink": "^4.3.1",
+        "openpgp": "github:larabr/openpgpjs#ts-update-stream-types-and-other-options",
```

Comment:
Declaring OpenPGP.js as non-dev dep is needed as long as we export utils functions which are in files that import `openpgp` modules. Otherwise webpack/TS in the apps fail to compile (before tree-shaking).

To avoid this, we should consider moving the util functions and constants to separate files or to the webapp repo directly.